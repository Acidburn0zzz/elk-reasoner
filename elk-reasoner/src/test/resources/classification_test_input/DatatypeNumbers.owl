Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(:=<http://www.semanticweb.org/Test.owl#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<http://www.semanticweb.org/Test.owl>

EquivalentClasses(:AG DataSomeValuesFrom(:prop owl:real))
EquivalentClasses(:AH DataSomeValuesFrom(:prop owl:rational))
EquivalentClasses(:AI DataSomeValuesFrom(:prop xsd:decimal))
EquivalentClasses(:AJ DataSomeValuesFrom(:prop xsd:integer))
EquivalentClasses(:AK DataSomeValuesFrom(:prop xsd:nonNegativeInteger))

EquivalentClasses(:BA DataHasValue(:prop "5"^^xsd:nonNegativeInteger))
EquivalentClasses(:BB DataHasValue(:prop "10/2"^^owl:rational))
EquivalentClasses(:BC DataHasValue(:prop "5.0"^^xsd:decimal))
EquivalentClasses(:BD DataHasValue(:prop2 "1"^^xsd:nonNegativeInteger))
EquivalentClasses(:BE DataHasValue(:prop "5.000000000000000000000000000000000000001"^^xsd:decimal))
EquivalentClasses(:BF DataHasValue(:prop "5"^^xsd:integer))
EquivalentClasses(:BG DataHasValue(:prop "0/1"^^owl:rational))
EquivalentClasses(:BH DataHasValue(:prop "0.1"^^xsd:decimal))
EquivalentClasses(:BI DataHasValue(:prop "1/10"^^owl:rational))
EquivalentClasses(:BJ DataHasValue(:prop "-3"^^xsd:integer))
EquivalentClasses(:BK DataHasValue(:prop "0"^^xsd:integer))
EquivalentClasses(:BL DataHasValue(:prop "-1.1"^^xsd:decimal))

EquivalentClasses(:CA DataSomeValuesFrom(:prop DatatypeRestriction(xsd:integer xsd:minExclusive "0"^^xsd:integer)))
EquivalentClasses(:CB DataSomeValuesFrom(:prop DatatypeRestriction(owl:real xsd:minInclusive "0.0"^^xsd:decimal xsd:maxExclusive "1/1"^^owl:rational)))
EquivalentClasses(:CC DataSomeValuesFrom(:prop DatatypeRestriction(xsd:integer xsd:maxExclusive "10"^^xsd:integer xsd:minExclusive "12"^^xsd:integer)))
EquivalentClasses(:CD DataSomeValuesFrom(:prop DatatypeRestriction(xsd:integer xsd:maxExclusive "5"^^xsd:integer xsd:minExclusive "4"^^xsd:integer)))
EquivalentClasses(:CE DataSomeValuesFrom(:prop DatatypeRestriction(owl:rational xsd:minInclusive "1.01"^^xsd:decimal)))
EquivalentClasses(:CF DataSomeValuesFrom(:prop DatatypeRestriction(xsd:nonNegativeInteger xsd:minInclusive "5.1"^^xsd:decimal xsd:maxExclusive "6"^^xsd:integer)))
EquivalentClasses(:CG DataSomeValuesFrom(:prop DatatypeRestriction(xsd:integer xsd:maxExclusive "5"^^xsd:integer)))
EquivalentClasses(:CH DataSomeValuesFrom(:prop DatatypeRestriction(xsd:nonNegativeInteger xsd:maxExclusive "5"^^xsd:integer)))

)