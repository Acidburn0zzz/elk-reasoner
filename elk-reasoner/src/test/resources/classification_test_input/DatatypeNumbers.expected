Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<http://www.semanticweb.org/Test.owl>

SubClassOf(<http://www.semanticweb.org/Test.owl#AG> owl:Thing)
SubClassOf(<http://www.semanticweb.org/Test.owl#BD> owl:Thing)
SubClassOf(<http://www.semanticweb.org/Test.owl#AH> <http://www.semanticweb.org/Test.owl#AG>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AI> <http://www.semanticweb.org/Test.owl#AH>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AJ> <http://www.semanticweb.org/Test.owl#AI>)
SubClassOf(<http://www.semanticweb.org/Test.owl#AK> <http://www.semanticweb.org/Test.owl#AJ>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BA> <http://www.semanticweb.org/Test.owl#CA>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BA> <http://www.semanticweb.org/Test.owl#CE>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BB> <http://www.semanticweb.org/Test.owl#CA>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BB> <http://www.semanticweb.org/Test.owl#CE>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BC> <http://www.semanticweb.org/Test.owl#CA>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BC> <http://www.semanticweb.org/Test.owl#CE>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BE> <http://www.semanticweb.org/Test.owl#AI>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BE> <http://www.semanticweb.org/Test.owl#CE>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BF> <http://www.semanticweb.org/Test.owl#CA>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BF> <http://www.semanticweb.org/Test.owl#CE>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BG> <http://www.semanticweb.org/Test.owl#CB>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BG> <http://www.semanticweb.org/Test.owl#CH>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BH> <http://www.semanticweb.org/Test.owl#AI>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BH> <http://www.semanticweb.org/Test.owl#CB>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BI> <http://www.semanticweb.org/Test.owl#AI>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BI> <http://www.semanticweb.org/Test.owl#CB>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BJ> <http://www.semanticweb.org/Test.owl#CG>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BK> <http://www.semanticweb.org/Test.owl#CB>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BK> <http://www.semanticweb.org/Test.owl#CH>)
SubClassOf(<http://www.semanticweb.org/Test.owl#BL> <http://www.semanticweb.org/Test.owl#AI>)
SubClassOf(<http://www.semanticweb.org/Test.owl#CA> <http://www.semanticweb.org/Test.owl#AK>)
SubClassOf(<http://www.semanticweb.org/Test.owl#CB> <http://www.semanticweb.org/Test.owl#AG>)
SubClassOf(<http://www.semanticweb.org/Test.owl#CE> <http://www.semanticweb.org/Test.owl#AH>)
SubClassOf(<http://www.semanticweb.org/Test.owl#CG> <http://www.semanticweb.org/Test.owl#AJ>)
SubClassOf(<http://www.semanticweb.org/Test.owl#CH> <http://www.semanticweb.org/Test.owl#AK>)
SubClassOf(<http://www.semanticweb.org/Test.owl#CH> <http://www.semanticweb.org/Test.owl#CG>)
EquivalentClasses(<http://www.semanticweb.org/Test.owl#BA> <http://www.semanticweb.org/Test.owl#BB> <http://www.semanticweb.org/Test.owl#BC> <http://www.semanticweb.org/Test.owl#BF>)
EquivalentClasses(<http://www.semanticweb.org/Test.owl#BA> <http://www.semanticweb.org/Test.owl#BB> <http://www.semanticweb.org/Test.owl#BC> <http://www.semanticweb.org/Test.owl#BF>)
EquivalentClasses(<http://www.semanticweb.org/Test.owl#BA> <http://www.semanticweb.org/Test.owl#BB> <http://www.semanticweb.org/Test.owl#BC> <http://www.semanticweb.org/Test.owl#BF>)
EquivalentClasses(<http://www.semanticweb.org/Test.owl#BA> <http://www.semanticweb.org/Test.owl#BB> <http://www.semanticweb.org/Test.owl#BC> <http://www.semanticweb.org/Test.owl#BF>)
EquivalentClasses(<http://www.semanticweb.org/Test.owl#BG> <http://www.semanticweb.org/Test.owl#BK>)
EquivalentClasses(<http://www.semanticweb.org/Test.owl#BH> <http://www.semanticweb.org/Test.owl#BI>)
EquivalentClasses(<http://www.semanticweb.org/Test.owl#BI> <http://www.semanticweb.org/Test.owl#BH>)
EquivalentClasses(<http://www.semanticweb.org/Test.owl#BK> <http://www.semanticweb.org/Test.owl#BG>)
EquivalentClasses(<http://www.semanticweb.org/Test.owl#CC> <http://www.semanticweb.org/Test.owl#CD> <http://www.semanticweb.org/Test.owl#CF> owl:Nothing)
EquivalentClasses(<http://www.semanticweb.org/Test.owl#CC> <http://www.semanticweb.org/Test.owl#CD> <http://www.semanticweb.org/Test.owl#CF> owl:Nothing)
EquivalentClasses(<http://www.semanticweb.org/Test.owl#CC> <http://www.semanticweb.org/Test.owl#CD> <http://www.semanticweb.org/Test.owl#CF> owl:Nothing)
EquivalentClasses(<http://www.semanticweb.org/Test.owl#CC> <http://www.semanticweb.org/Test.owl#CD> <http://www.semanticweb.org/Test.owl#CF> owl:Nothing)
)
